### PR TITLE
Rev the controller manager version to pick up machine deployment controller

### PR DESCRIPTION
**Pull request by k4leung4**
**What this PR does / why we need it**:
This is to pick up PR #143, which allows for the creation of machine
deployments. Rolling update should now work.

This also updates the apiserver to be the same as the one used for gce and terraform provider

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note

```

@kubernetes/kube-deploy-reviewers

### CODE REVIEW

File: cloud/vsphere/pods.go [hunk 1]
```diff
@@ -32,8 +32,8 @@ import (
 	"sigs.k8s.io/cluster-api/cloud/vsphere/config"
 )
 
-var apiServerImage = "gcr.io/k8s-cluster-api/cluster-apiserver:0.0.4"
-var controllerManagerImage = "gcr.io/k8s-cluster-api/controller-manager:0.0.4"
+var apiServerImage = "gcr.io/k8s-cluster-api/cluster-apiserver:0.0.5"
```
Comment (kcoronado) [inline cloud/vsphere/pods.go]: Did you mean to change the apiserver version for just vsphere? Would you need to update the makefile too?

Comment (k4leung4) [reply cloud/vsphere/pods.go]: updated the apiserver as that is what is being used for gce and terraform.

Comment (kcoronado) [reply cloud/vsphere/pods.go]: Oh I didn't see the others were already at 0.0.5, oops.

**Pull request merged**